modules/rosa-sts-creating-roles.adoc

```diff
@@ -0,0 +1,624 @@
+                "elasticloadbalancing:AttachLoadBalancerToSubnets",
+                "elasticloadbalancing:ApplySecurityGroupsToLoadBalancer",
+                "elasticloadbalancing:CreateListener",
+                "elasticloadbalancing:CreateLoadBalancer",
+                "elasticloadbalancing:CreateLoadBalancerPolicy",
+                "elasticloadbalancing:CreateLoadBalancerListeners",
+                "elasticloadbalancing:CreateTargetGroup",
+                "elasticloadbalancing:ConfigureHealthCheck",
+                "elasticloadbalancing:DeleteListener",
+                "elasticloadbalancing:DeleteLoadBalancer",
+                "elasticloadbalancing:DeleteLoadBalancerListeners",
+                "elasticloadbalancing:DeleteTargetGroup",
+                "elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
+                "elasticloadbalancing:DeregisterTargets",
+                "elasticloadbalancing:Describe*",
+                "elasticloadbalancing:DetachLoadBalancerFromSubnets",
+                "elasticloadbalancing:ModifyListener",
+                "elasticloadbalancing:ModifyLoadBalancerAttributes",
+                "elasticloadbalancing:ModifyTargetGroup",
+                "elasticloadbalancing:ModifyTargetGroupAttributes",
+                "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+                "elasticloadbalancing:RegisterTargets",
+                "elasticloadbalancing:SetLoadBalancerPoliciesForBackendServer",
+                "elasticloadbalancing:SetLoadBalancerPoliciesOfListener",
+                "kms:DescribeKey"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+EOM
+----
+====
+. Attach the inline policy in the [filename]`ManagedOpenShift_ControlPlane_Role_Policy.json` file to the `ManagedOpenShift-ControlPlane-Role` instance profile role.
++
+[source, terminal]
+----
+$ aws iam put-role-policy --role-name ManagedOpenShift-ControlPlane-Role --policy-name -ManagedOpenShift-ControlPlane-Role-Policy --policy-document file://ManagedOpenShift_ControlPlane_Role_Policy.json
+----
+
+== Worker node instance profile role
+
+. Write the trust policy for the `ManagedOpenShift-Worker-Role` access instance profile role to the [filename]`ManagedOpenShift_Worker_Role.json` file.
++
+.Command to write the trust policy
+[%collapsible]
+====
+[source,terminal]
+----
+$ cat
+cat <<EOM > ManagedOpenShift_Worker_Role.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"       
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOM
+----
+====
+. Create the `ManagedOpenShift-Worker-Role` role with the trust policy using the AWS CLI.
++
+[source,terminal]
+----
+$ aws iam create-role --role-name ManagedOpenShift-Worker-Role --assume-role-policy-document file://ManagedOpenShift_Worker_Role_Policy.json [--permissions-boundary ${permissions_boundary_arn}]
```

Comment:
same issue with above.
File named created is ManagedOpenShift_Worker_Role.son
But used file://ManagedOpenShift_Worker_Role_Policy.json